src/Tizen.NUI/src/public/Common/Position.cs

```diff
@@ -29,6 +29,8 @@ namespace Tizen.NUI
     [Tizen.NUI.Binding.TypeConverter(typeof(PositionTypeConverter))]
     public class Position : Disposable, ICloneable
     {
+        private static readonly Vector3 one = new Vector3(1.0f, 1.0f, 1.0f);
+        private static readonly Vector3 zero = new Vector3(0.0f, 0.0f, 0.0f);
```

Comment:
```suggestion
        private static readonly Position one = new Position(1.0f, 1.0f, 1.0f);
        private static readonly Position zero = new Position(0.0f, 0.0f, 0.0f);
```

Don't we need to use `Position` class for here, instead `Vector3`?

Reply:
Thank you for suggestion